### PR TITLE
Change the names of artifacts created by goreleaser

### PR DESCRIPTION
**Pull request by jrcast**
Change the names of artifacts created by goreleaser
New names:
image

### CODE REVIEW

**Comment by jrcast** [comment]
This is the new list of artifacts produced:

image

**Review by lee-baines — APPROVED** [review]

**Review by dgoldssfo — APPROVED** [review]
This comment is a little out of scope for this PR, but the image triggers a question for me - why are we producing a Windows executable when we don't support Windows in any other way?

Might having one lead to some confusion that you can actually run our stuff on Kubernetes on Windows? We're currently telling Windows users they have to run on a Linux VM. (Plan to change this to WSL2 when it's finally ready for prime time.)

**Review by wstrange — APPROVED** [review]

**Comment by maxres-fr** [comment]
> Might having one lead to some confusion that you can actually run our stuff on Kubernetes on Windows? We're currently telling Windows users they have to run on a Linux VM. (Plan to change this to WSL2 when it's finally ready for prime time.)

I think I had left it in thinking we might be able to support with little to no effort -- however I think somewhere there might be ticket to remove. but Maybe @jrcast  can just tackle that here too.

File: .goreleaser.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ archives:
       linux: Linux
       windows: Windows
       amd64: x86_64
+    name_template:  "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```
Comment (maxres-fr) [inline .goreleaser.yml:27]: so nice GH wont let me comment on the line that isn't changed in the PR...

could you remove "windows" from `goos` key? https://github.com/ForgeRock/forgeops-cli/pull/46/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R15

**Review by maxres-fr — APPROVED** [review]

File: .goreleaser.yml (hunk 1 above)
Comment (maxres-fr) [reply .goreleaser.yml:27]: and that link doesn't work either ~Loc 25

Comment (jrcast) [reply .goreleaser.yml:27]: I think we should also remove `arm` from goarch. I don't think we're going to see much arm7 action with this tool

**Pull request merged**